analysis/tests/src/expected/DocExtractionRes.res.txt

```diff
@@ -127,7 +127,18 @@ extracting docs for src/DocExtractionRes.res
         {
           "name": "SomeStuff",
           "docstrings": ["This has inline records..."],
-          "signature": "SomeStuff"
+          "signature": "SomeStuff({offline: bool, online?: bool})",
+          "inlineRecordFields": [{
+            "name": "offline",
+            "optional": false,
+            "docstrings": [],
+            "signature": "bool"
+          }, {
```

Comment:
I think is better for the field to be named `items`. A variant payload can be a tuple and we use `items` elsewhere

```jsonc
{
  "name": "SomeStuff",
  "items": [{...}],
}
```